keras_cv/models/object_detection/faster_rcnn.py

```diff
@@ -388,12 +388,17 @@ def compile(
         rpn_classification_loss=None,
         weight_decay=0.0001,
         loss=None,
+        metrics=None,
```

Comment:
If we currently don't support it, then no need to expose it.

Reply:
I did this so that `metrics=[]` can pass

Reply:
updated to remove